packages/ui-react/src/components/Shelf/Shelf.module.scss

```diff
@@ -38,22 +54,31 @@
   outline-color: var(--highlight-color, white);
   cursor: pointer;
   transition: transform 0.3s ease-out, opacity 0.3s ease-out;
+  appearance: none;
 
   > svg {
     width: 30px;
     height: 30px;
   }
-  &.disabled {
-    cursor: default;
-    &:hover {
-      transform: none;
-    }
+
+  &:disabled {
+    opacity: 0.3;
+    pointer-events: none;
```

Comment:
Currently the disabled attribute and class is not present anymore. So this is unused CSS.
Should we reintroduce these classes? Previously it was previously only present for the prev-button (and as you said; it caused e2e failures).